### PR TITLE
Opens GitHub's homepage if there is no notification

### PR DESCRIPTION
**Pull request by Narno**
1. saves the notifications count in local storage
2. checks the notifications count value when cliking on icon
3. if the notifications count value is > 0, opens Github notifications page otherwise opens Github homepage

### CODE REVIEW

File: extension/main.js [hunk 1]
```diff
@@ -73,19 +75,23 @@
...
 	chrome.browserAction.onClicked.addListener(function (tab) {
 		var url = window.GitHubNotify.settings.get('rootUrl');
-
 		if (/api.github.com\/$/.test(url)) {
 			url = 'https://github.com/';
 		}
 
-		var notifTab = {
-			url: url + 'notifications'
+		var GitHubTab = {
```
Comment (sindresorhus) [inline extension/main.js]: `var ghTab = {`

File: extension/main.js [hunk 2]
```diff
@@ -73,19 +75,23 @@
...
+			url: url
 		};
+		if (window.GitHubNotify.settings.get('count') > 0) {
+			GitHubTab = {
+				url: url + 'notifications'
+			};
```
Comment (sindresorhus) [inline extension/main.js]: `ghTab.url  = url + 'notifications';`

**Comment by sindresorhus** [comment]
Good idea! Thank you.

**Comment by Narno** [comment]
With pleasure! :-)

**Pull request closed without merging**